src/components/SharedComponents/DateDisplay.js

```diff
@@ -8,16 +8,19 @@ import IconMaterial from "react-native-vector-icons/MaterialIcons";
 import { formatApiDatetime } from "sharedHelpers/dateAndTime";
 
 type Props = {
+  pretext?: string,
   dateString: String
 };
 
 const DateDisplay = ( {
-  dateString
+  dateString,
+  pretext
 }: Props ): React.Node => {
   const { t } = useTranslation( );
   return (
     <View className="flex flex-row items-center">
       <IconMaterial name="watch-later" size={15} />
+      {pretext && <Body4 className="ml-[5px]">{ pretext }</Body4>}
```

Comment:
I would rather combine the strings with a whitespace in the lower Body4.

Reply:
Also text-darkGray is redundant now in the lower one, because it is default in Body4.